src/evaluator/expression/quote.cpp

```diff
@@ -32,7 +32,7 @@ Quote::parse(Span span, std::shared_ptr<List> arguments) {
     return std::make_unique<Quote>(Quote(span, element));
 }
 
-std::shared_ptr<Element> Quote::evaluate() const { return this->element; }
+std::shared_ptr<Element> Quote::evaluate(std::shared_ptr<Scope> parent) const { return this->element; }
```

Comment:
```suggestion
std::shared_ptr<Element> Quote::evaluate(std::shared_ptr<Scope>) const { return this->element; }
```

иначе будет ворнинг о неиспользуемой переменной. а тут скоуп точно не нужен